docs/sources/articles/networking.md

```diff
@@ -599,7 +599,8 @@ options are configurable at server startup:
 
  *  `--bip=CIDR` — supply a specific IP address and netmask for the
     `docker0` bridge, using standard CIDR notation like
-    `192.168.1.5/24`.
+    `192.168.1.5/24`. If the passing CIDR is not the same with the current
+    bridge IP address, docker will override the old IP with the new CIDR.
```

Comment:
Please use "If the provided CIDR does not match the current" on the first line.

Second line: ".. , docker will replace the current bridge IP with the provided CIDR."

Thanks!